backend/src/services/bambora.js

```diff
@@ -41,12 +46,12 @@ module.exports = {
             },
             products: [
                 {
-                    id: '1',
-                    title: 'Vanta_App',
+                    id: '327554_163423301_MF',
```

Comment:
this should belongs to the environmental variables

Reply:
moved to environment variables